create_aio_app/main.py

```diff
@@ -15,38 +15,47 @@
 echo = partial(click.echo, err=True)
 
 
+def show_commands(folder):
+    try:
+        os.chdir(f"{folder}/")  # nosec
+        os.system(f"make help")  # nosec
+    except Exception as e:
+        echo(f'{e}')
+        echo(click.style("\nFailed to show commands\n", fg="red",))
+
+
 def main():
     args = parse_arguments()
-    template_path = str(parent / 'template')
+    template_path = str(parent / "template")
 
     kwargs = {}
 
-    if args.get('name'):
+    if args.get("name"):
         kwargs = {
-            'no_input': True,
-            'extra_context': {
-                'project_name': args.get('name'),
-                'use_postgres': 'n' if args.get('without_postgres') else 'y',
-                'use_redis': 'y' if args.get('redis') else 'n',
-                'use_uvloop': 'y' if args.get('uvloop') else 'n',
+            "no_input": True,
+            "extra_context": {
+                "project_name": args.get("name"),
+                "use_postgres": "n" if args.get("without_postgres") else "y",
+                "use_redis": "y" if args.get("redis") else "n",
+                "use_uvloop": "y" if args.get("uvloop") else "n",
             },
         }
 
     try:
         result = cookiecutter(template_path, **kwargs)
     except (FailedHookException, OutputDirExistsException) as exc:
         if isinstance(exc, OutputDirExistsException):
-            echo(click.style(
-                '\n\nDirectory with such name already exists!\n',
-                fg='red',
-            ))
+            echo(
+                click.style(
+                    "\n\nDirectory with such name already exists!\n", fg="red")
+            )
         return
 
     folder = Path(result).name
 
-    echo(click.style(
-        '\n\nSuccessfully generated!\n',
-        fg='bright_green',
-    ))
-    echo('cd ' + click.style(f'{folder}/', fg='bright_blue'))
-    echo('make run\n\n')
```

Comment:
a'u sure that this line is important here?

Reply:
I'm not sure if the line was the same one, cause it seems that the main file you're referencing is different from the main I've modified. Not sure if that's a problem.